test/functional/toolchains/qbs/test_qbs.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from conan.internal.api.new.cmake_lib import source_cpp, source_h
+from conan.internal.api.new.qbs_lib import qbs_file, conanfile_sources
+from conan.test.utils.tools import TestClient
+from jinja2 import Template
+
+
+def gen_file(template, **context):
+    t = Template(template)
+    return t.render(**context)
+
+
+@pytest.mark.tool("qbs")
+def test_qbs_static_lib():
```

Comment:
It is not necessary to test the template files themselves.
Testing that the ``conan new qbs_lib`` actually works is already covering this, and is necessary in any case, so I think this test can be removed.